src/mongoose-softdelete.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Schema } from "mongoose";
+
+export default function (schema: Schema<any>): void;
```

Comment:
this definition can be part of the `index.d.ts` I think. The package declares the type in `package.json` from `index.d.ts` only

Reply:
You're right. I changed it